src/overlays/actors/ovl_En_Ex_Ruppy/z_en_ex_ruppy.c

```diff
@@ -47,12 +48,12 @@ void EnExRuppy_Init(Actor* thisx, GlobalContext* globalCtx) {
     f32 temp2;
     s16 temp3;
 
-    this->unk_152 = this->actor.params;
-
-    osSyncPrintf(VT_FGCOL(GREEN) "☆☆☆☆☆ インデックス ☆☆☆☆☆ %x\n" VT_RST, this->unk_152); // "Index"
+    this->type = this->actor.params;
+    // Index
+    osSyncPrintf(VT_FGCOL(GREEN) "☆☆☆☆☆ インデックス ☆☆☆☆☆ %x\n" VT_RST, this->type); // "Index"
```

Comment:
It looks like there are quite a few duplicated comments in this file now
```suggestion
    osSyncPrintf(VT_FGCOL(GREEN) "☆☆☆☆☆ インデックス ☆☆☆☆☆ %x\n" VT_RST, this->type); // "Index"
```

Reply:
i also do recommend putting translation comments in quotes like the original one there does